modules/container-insights/main.tf

```diff
@@ -8,7 +8,7 @@ locals {
 module "irsa" {
   source         = "../iam-role-for-serviceaccount"
   count          = var.enabled ? 1 : 0
-  name           = join("-", ["irsa", local.name])
+  name           = substr(join("-", ["irsa", local.name]), 0, 64)
```

Comment:
Suggestion, how do you think of applying this type restriction setting to the underlying code? This *irsa* module utilizes submodules of the same eks module.

https://github.com/Young-ook/terraform-aws-eks/blob/f4fddcb7012d3993a3e931be0e346f81c5792544/modules/iam-role-for-serviceaccount/labels.tf#L12

Reply:
Ok, I will try to update that way and test it on my environment.
thanks for the suggestion.